addons/mail/static/src/owl/store/messaging_actions.js

```diff
@@ -236,6 +254,35 @@ const actions = {
             dispatch('openThread', threadLocalId);
         }
     },
+    /**
+     * @param {Object} param0
+     * @param {function} param0.dispatch
+     * @param {Object} param0.getters
+     * @param {Object} param0.state
+     * @param {Object} param1
+     * @param {string} param1.initialThreadId
+     * @param {string} param1.initialThreadModel
+     * @return {string}
+     */
+    createChatter({ dispatch, getters, state }, { initialThreadId, initialThreadModel }) {
+        const chatterLocalId = _.uniqueId('o_Chatter');
+        const chatter = {
+            isAttachmentBoxVisible: false,
+            isComposerLog: true,
+            isComposerVisible: false,
+            isDisabled: !initialThreadId,
+            localId: chatterLocalId,
+            threadId: initialThreadId,
+            threadLocalId: undefined,
+            threadModel: initialThreadModel,
+        };
+        state.chatters[chatterLocalId] = chatter;
+        dispatch('_computeChatter', chatterLocalId, {
+            threadId: initialThreadId,
+            threadModel: initialThreadModel,
```

Comment:
remove extra args: not used, and compute links will change with store entities + auto-computation of links :)